content/blog/cruxbox/2020-08-06_gsoc_2020_project_xfs_project_update.md

```diff
@@ -0,0 +1,46 @@
++++
+type = "blog"
+title = "GSoC 2020 Project Update: XFS"
+author = "CruxBox"
+date = "2020-11-08 17:58:00+05:30"
+tags = ["haiku", "software", "gsoc", "gsoc2020", "file system"]
++++
+
+Hello there!
+There is quite a lot to share about the work done this month.
+---
+
+Work done so far:
+You can check for the patches here: https://review.haiku-os.org/q/owner:shubhambhagat111%2540yahoo.com
+In short: All directory work is done. This includes ShortForm Directories, Block Directories, Leaf Directories, Node Directories and B+Tree based Directories.
+
+Detailed view at the work:
+Some context: Leaf entries containt hash values of a directory entry name and also the address of that entry. We use this for fast lookups.
+1) Leaf Directories:
```

Comment:
Add a blank line in front to start the numbered list in a new paragraph.

Reply:
Ack